src/tools/chip-cert/CertUtils.cpp

```diff
@@ -260,23 +275,26 @@ CertFormat DetectCertFormat(const uint8_t * cert, uint32_t certLen)
     return kCertFormat_Unknown;
 }
 
-bool SetCertSerialNumber(X509 * cert)
+bool SetCertSerialNumber(X509 * cert, uint64_t value = kUseRandomSerialNumber)
 {
     bool res = true;
     uint64_t rnd;
     ASN1_INTEGER * snInt = X509_get_serialNumber(cert);
 
-    // Generate a random value to be used as the serial number.
-    if (!RAND_bytes(reinterpret_cast<uint8_t *>(&rnd), sizeof(rnd)))
+    if (value == kUseRandomSerialNumber)
     {
-        ReportOpenSSLErrorAndExit("RAND_bytes", res = false);
-    }
+        // Generate a random value to be used as the serial number.
+        if (!RAND_bytes(reinterpret_cast<uint8_t *>(&rnd), sizeof(rnd)))
```

Comment:
Suggest using `DRBG_get_bytes` from CHIPCryptoPAL instead of RAND_bytes while we're here (assuming crypto is initialized by chip-cert tool...)